db/migrate/20170225203658_add_user_id_to_spree_credit_cards.rb

```diff
@@ -0,0 +1,13 @@
+class AddUserIdToSpreeCreditCards < ActiveRecord::Migration
+  def change
+    unless Spree::CreditCard.column_names.include? "user_id"
+      add_column :spree_credit_cards, :user_id, :integer
+      add_index :spree_credit_cards, :user_id
+    end
+
+    unless Spree::CreditCard.column_names.include? "payment_method_id"
+      add_column :spree_credit_cards, :payment_method_id, :integer
+      add_index :spree_credit_cards, :payment_method_id
+    end
+  end
+end
```

Comment:
Sorry, I know this is a pain, but could we split these into two separate migrations?